penaltymodel_cache/setup.py

```diff
@@ -53,7 +53,6 @@
     extras_require=extras_require,
     classifiers=classifiers,
     python_requires=python_requires,
-    entry_points={'penaltymodel_factory': ['maxgap = penaltymodel.cache:get_penalty_model'],
-                  'penaltymodel_cache': ['penaltymodel_cache = penaltymodel.cache:cache_penalty_model']},
+    entry_points={'penaltymodel_cache': ['penaltymodel_cache = penaltymodel.cache:cache_penalty_model']},
```

Comment:
I think you meant

```
 entry_points={'penaltymodel_factory': ['penaltymodel_cache = penaltymodel.cache:get_penalty_model'],
                        'penaltymodel_cache': ['penaltymodel_cache = penaltymodel.cache:cache_penalty_model']},
```

Reply:
or even
```
    entry_points={'penaltymodel_factory': ['cache = penaltymodel.cache:get_penalty_model'],
                  'penaltymodel_cache': ['cache = penaltymodel.cache:cache_penalty_model']},
```

Reply:
Yes.

Although, I would like to separate actual factories from caching layer. Maybe we could have `penaltymodel_upstreamfactory`, `penaltymodel_cache` and `penaltymodel_factory` (which would include both).
Or, alternative naming: `penaltymodel_factory`, `penaltymodel_cache`, `penaltymodel_cachedfactory`?